pkg/monitor/api/types_test.go

```diff
@@ -4,6 +4,8 @@
 package api
```

Comment:
I'd rename commit `cleanup: use MonitorFormatter to log unknown events` -> `monitor: use MonitorFormatter to log unknown events` so that it is more immediate to retrieve the unit it applies to.